src/app/code/community/Omise/Gateway/Model/Event/Charge/Complete.php

```diff
@@ -50,12 +50,14 @@ public function handle($event)
             return;
         }
 
-        if ($order->isPaymentReview() && ($charge->isSuccessful() || $charge->isAwaitCapture())) {
+        $isPendingOrReview = $order->isPaymentReview() || $order->isPaymentPending();
```

Comment:
extremely small thing but maybe `$isPaymentProcessing` name would be better?

Reply:
Not sure because at this point the payment has either been cancelled (i.e. still pending), or has happened (needs review). In neither of these 2 cases is it actually being processed?